.github/workflows/integration-test-machine.yaml

```diff
@@ -26,6 +26,10 @@ jobs:
       - name: Setup operator environment
         run: |
           sudo snap install concierge --classic
+
+          # Workaround for lxd sometimes failing to refresh
+          sudo snap info lxd | grep "tracking": | awk '{print $2}' | grep "5.21/stable" || sudo snap stop lxd
+
```

Comment:
Should we open an issue in concierge to inform Jon that we have to go through this workaround?

Reply:
I found a relatively clean way of implementing the workaround on concierge. Here are the issue and PR:

https://github.com/jnsgruk/concierge/issues/20
https://github.com/jnsgruk/concierge/pull/21

If they get merged soon, we could drop this PR. Otherwise, we can merge this to unblock CI and revert it once it is not necessary.